### PR TITLE
feat: Update submodules (SC-4407)

### PR DESCRIPTION
**Pull request by JGcarv**

### CODE REVIEW

File: contracts/test/ParityTest.t.sol [hunk 1]
```diff
@@ -36,9 +36,11 @@ import { AddressRegistry } from "../AddressRegistry.sol";
 
 contract ParityTest is AddressRegistry, StateManipulations, TestUtils {
 
-    uint256 constant WAD = 10 ** 18;  // ETH  precision
-    uint256 constant BTC = 10 ** 8;   // WBTC precision
-    uint256 constant USD = 10 ** 6;   // USDC precision
+    uint256 constant WAD      = 10 ** 18;  // ETH  precision
+    uint256 constant BTC      = 10 ** 8;   // WBTC precision
+    uint256 constant USD      = 10 ** 6;   // USDC precision
+    uint256 constant ZERO     = uint256(0);
```
Comment (lucas-manuel) [inline contracts/test/ParityTest.t.sol:42]: Was this needed?

File: contracts/test/ParityTest.t.sol [hunk 2]
```diff
@@ -550,6 +552,7 @@ contract ParityTest is AddressRegistry, StateManipulations, TestUtils {
                 address(debtLocker.liquidator()), 
                 10 * BTC, 
                 type(uint256).max,
+                uint256(0),
```
Comment (lucas-manuel) [inline contracts/test/ParityTest.t.sol:555]: If so lets add here for consistency

File: contracts/test/ParityTest.t.sol [hunk 3]
```diff
@@ -641,10 +659,10 @@ contract ParityTest is AddressRegistry, StateManipulations, TestUtils {
         /*****************/
         /*** Fund Loan ***/
         /*****************/
-
+
```
Comment (lucas-manuel) [inline contracts/test/ParityTest.t.sol:662]: Rm whitespace

**Review by lucas-manuel — CHANGES_REQUESTED** [review]

File: contracts/test/ParityTest.t.sol (hunk 1 above)
Comment (JGcarv) [reply contracts/test/ParityTest.t.sol:42]: It was an attempt to remove items from the stack, but I didn't help. I do think is nice though

**Review by lucas-manuel — APPROVED** [review]

**Pull request merged**